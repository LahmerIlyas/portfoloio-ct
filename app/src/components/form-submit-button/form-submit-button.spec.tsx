import React from 'react';
import renderer from 'react-test-renderer';
import { Formik } from 'formik';
import { FormSubmitButton } from '.';

describe('<FormSubmitButton />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Formik initialValues={{}} onSubmit={() => {}}>
          <FormSubmitButton title="Apply" />
        </Formik>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
