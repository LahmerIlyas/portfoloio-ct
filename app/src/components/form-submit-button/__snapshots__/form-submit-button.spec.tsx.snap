// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormSubmitButton /> renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "rgb(145, 199, 136)",
        "borderRadius": 3,
        "flexDirection": "row",
        "justifyContent": "center",
        "paddingHorizontal": 22,
        "paddingVertical": 16,
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Signika-Medium",
        "fontSize": 22,
      }
    }
  >
    Apply
  </Text>
</View>
`;
